src/extensions/rageclick.js

```diff
@@ -0,0 +1,32 @@
+const RAGE_CLICK_THRESHOLD = 30
```

Comment:
Maybe worth clarifying what each param is intended for in case someone else needs to understand this?